ui/packages/catalog-ui-search/src/main/webapp/component/editable-rows/editable-rows.view.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+let Backbone = require('backbone');
+let Marionette = require('marionette');
+let _ = require('underscore');
+let $ = require('jquery');
+let CustomElements = require('js/CustomElements');
+
+let RowView = Marionette.LayoutView.extend({
+    className: 'editable-rows-item',
+    template: '<div class="embed"></div>' +
```

Comment:
✏️ Most of the rest of Intrigue has templates defined as separate `editable-rows.hbs` type files in the same directory as opposed to this JSX-esque stuff.